TACC/Core-Styles
# feat: build has-dark-logo–themed stylesheets

## Overview

Build `has-dark-logo` theme stylesheets.

## Related

- required by https://github.com/TACC/Core-CMS/pull/607

## Changes

- add build commands for each file that uses `env()`

## Testing

0. Open this branch.
1. Delete these files:
    - `/dist/core-styles.header.theme--has-dark-logo.css`
    - `/dist/trumps/s-header.theme--has-dark-logo.css`
    - `/dist/trumps/tacc-search-bar.theme--has-dark-logo.css`
3. Run `npm run build`.
4. Verify the files you built are restored.
5. Compare the files deleted/built to:
    - `/dist/core-styles.header.css`
    - `/dist/trumps/s-header.css`
    - `/dist/trumps/tacc-search-bar.css`
6. See the changes are the values used to replace the `env(--…)` instances in:
    - `/src/lib/_imports/core-styles.header.css`
    - `/src/lib/_imports/trumps/s-header.css`
    - `/src/lib/_imports/trumps/tacc-search-bar.css`

## UI

See https://github.com/TACC/Core-CMS/pull/607.
